compiler/rustc_infer/src/infer/type_variable.rs

```diff
@@ -181,6 +260,11 @@ impl<'tcx> TypeVariableTable<'_, 'tcx> {
         self.storage.eq_relations.with_log(self.undo_log)
     }
 
+    #[inline]
+    fn sub_relations(&mut self) -> super::UnificationTable<'_, 'tcx, TyVidSubKey> {
```

Comment:
i guess same kinda thing here... sub relations makes me think of something else than what this is? no idea what a better name is, also aware that this was the name we had before we removed it :>